src/mame/heathkit/h89.cpp

```diff
@@ -258,20 +265,20 @@ uint8_t h89_state::m1_r(offs_t offset)
     Console I/O              | E8-EF | 350-357
     NMI                      | F0-F1 | 360-361
     General purpose port     | F2    | 362
+    Cassette I/O(MTR-88)     | F8-F9 | 370-371
     NMI                      | FA-FB | 372-373
 */
-void h89_state::h89_io(address_map &map)
+void h89_state::h89_base_io(address_map &map)
 {
 	map.unmap_value_high();
 	map.global_mask(0xff);
 
 	// Disk I/O #1 - 0170-0173 (0x78-0x7b)
 	//   Options
-	//     - Cassette I/O (only uses 0x78 - 0x79) - Requires MTR-88 ROM
 	//     - H37 5-1/4" Soft-sectored Controller - Requires MTR-90 ROM
 	//     - H47 Dual 8" Drives - Requires MTR-89 or MTR-90 ROM
 	//     - H67 8" Hard disk + 8" Floppy Drives - Requires MTR-90 ROM
-	map(0x78, 0x7b).rw(m_h37, FUNC(heath_z37_fdc_device::read), FUNC(heath_z37_fdc_device::write));
+	// map(0x78, 0x7b)
```

Comment:
This now duplicates the comment in the H89 address map – should it really be here as well?